heralding/honeypot.py

```diff
@@ -59,13 +60,26 @@ def __init__(self, config, loop):
                 logger.warning('Could not request public ip from ipify, error: %s', ex)
             await asyncio.sleep(3600)
 
+    def setup_wordlist(self):
+        # load wordlist in memory
+        try:
+            wordlist_file = self.config['hash_cracker']['wordlist_file']
```

Comment:
This will fail on default installs, where wordlist.txt would reside in the package directory - but you look in CWD. You need to do it the same way as we handling heralding.yml.

Reply:
Okay